execution-engine/src/engines/wasmi.rs

```diff
@@ -106,11 +102,11 @@ impl MemoryHandler for MemoryRef {
         })))
     }
 
-    fn get_slice_mut<'a>(
-        &'a mut self,
```

Comment:
What change resulted in the ability for us not to use lifetimes here?

Reply:
The compiler warns us to remove it. I believe it probably automatically adds it now.